docs/RELEASE-NOTES.rst

```diff
@@ -7,6 +7,13 @@ next-release
 Bug Fixes
 `````````
 
+v1.5.0
+------
+
+Added Functionality
+```````````````````
+* Support for virtual server source address translation configuration.
+
```

Comment:
Once we have the documentation in f5-ci-docs, we should probably add a pointer in here to that section (much like the Support for `Flannel VxLAN in Kubernetes`_ section). This is good for now.